articles/service-fabric/service-fabric-get-started-containers-linux.md

```diff
@@ -190,6 +190,9 @@ Specify the port mapping in the appropriate format. For this article, you need t
    </ServiceManifestImport>
 ```	
 
+Rererence [link](

```

Comment:
Typo: Reference

Reply:
>>  to encrypt credentials and configure cluster-wide credentials
Instead, refer to this as " on how to specify a default container repository for the cluster and how to use it in applications."